techniques/systemSettings/networking/firewall/1.0/firewall.st

```diff
@@ -0,0 +1,101 @@
+# Here we have our firewall configuration (iptables or nftables)
+# But format may not be exactly what the output of iptables-save or nft export.
+# What we will do it to maintain an export from a previous run to be able to compare
+# with current export and detect changes.
+# This way we will be able to report config diffs and fix the config when necessary.
+
+bundle agent rudder_firewall {
+  vars:
+      "config"    string => "&FIREWALL_CONF&";
+      "firewall"  string => "rudder_&FIREWALL_IMPL&";
+      "posthook"  string => "&FIREWALL_POST_HOOK_COMMAND&";
+      "reference" string => "/var/rudder/tmp/firewall.ref";
+
+  methods:
+      "any" usebundle => rudder_set_force_component_none("true");
+      "en1" usebundle => enable_reporting;
+      # Package
+      "any" usebundle => _method_reporting_context("Installation", "None");
+      "any" usebundle => package_present("${${firewall}.package}", "", "", "");
+      # Service
+      "any" usebundle => _method_reporting_context("Service activation", "None");
```

Comment:
i'm not sure, but i think service ctivation should only be if !rudder_firewall_broken_template

Reply:
done